### PR TITLE
Feature/contact service

### PR DESCRIPTION
**Pull request by Slavitor**
Contact Us interactor, Interactor implementation, Contact us model and Contact us interactor tests.

### CODE REVIEW

File: Showcase-iOS/Enum/Path.swift [hunk 1]
```diff
@@ -9,7 +9,7 @@
 import Foundation
 
 enum Path: String {
-    case contacts
-    case apps
+    case contacts = "offices"
```
Comment (pjwelcome) [inline Showcase-iOS/Enum/Path.swift]: this case should be named offices

File: Showcase-iOSTests/Contact Us/ContactUsInteractableTests.swift [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        stub(mockDatabaseRefeceabele) { mock in
+            let _ = when(mock.observe(eventType: any(), with: any(), withCancel: any()).then({ _, _, errorCompletion in
+                errorCompletion(self.mockError!)
+            }))
+        }
+        stub(mockContactUsPresentable) { mock in
+            _ = when(mock.onRetrieveOfficesFailed(with: any()).then({ error in
+                let errorResult = error as NSError
+                XCTAssertEqual(errorResult, self.mockError!)
+                return
```
Comment (pjwelcome) [inline Showcase-iOSTests/Contact Us/ContactUsInteractableTests.swift]: why are you returning here ?

**Review by pjwelcome — CHANGES_REQUESTED** [review]

File: Showcase-iOSTests/Contact Us/ContactUsInteractableTests.swift (hunk 2 above)
Comment (Slavitor) [reply Showcase-iOSTests/Contact Us/ContactUsInteractableTests.swift]: will fix them. Thanks

File: Showcase-iOS/Contact Us/Model/Office.swift [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    var googleMapsName = ""
+    var googleMapsPlaceId = ""
+    var image = ""
+    var latitude = ""
+    var longitude = ""
+    var name = ""
+    var telephone = ""
+    
+    init(with snapShot: DataSnapshot?) {
+        if let snapShotValue = snapShot?.value as? [String: Any] {
```
Comment (pjwelcome) [inline Showcase-iOS/Contact Us/Model/Office.swift:18]: This needs to be tested in the Next PR Please

**Review by pjwelcome — APPROVED** [review]

**Pull request merged**